web-components/src/components/combobox/scss/combobox.scss

```diff
@@ -172,13 +172,13 @@ ul {
     font-weight: 400;
     font-size: 14px;
     line-height: 19px;
-    color: var(--combobox-group-label-color);
+    color: var( --combobox-groupLabel-color);
     padding: 9px 16px;
-    border-top: 1px solid var(--combobox-group-label-border-color);
-    border-bottom: 1px solid var(--combobox-group-label-border-color);
+    border-top: 1px solid var(--combobox-groupLabel-border-color);
```

Comment:
why did we change to camelcase (groupLabel)? ,

Reply:
@merajcal Fixed